Tests/FoundationTests/NSAttributedStringExtensionsTests.swift

```diff
@@ -70,6 +70,23 @@ final class NSAttributedStringExtensionsTests: XCTestCase {
     }
 
     // MARK: - Methods
+    func testApplying() {
+        #if canImport(AppKit) || canImport(UIKit)
+        let string = NSAttributedString(string: "Applying")
+        let out = string.applying(attributes: [
+            .strikethroughStyle: NSNumber(value: NSUnderlineStyle.single.rawValue as Int),
+            .foregroundColor: Color.red
+        ])
+        let attributes = out.attributes(at: 0, effectiveRange: nil)
+        let filteredAttributes = attributes.filter { (key, value) -> Bool in
+            return (key == NSAttributedString.Key.foregroundColor && (value as? Color) == .red) ||
+                (key == NSAttributedString.Key.strikethroughStyle && (value as? NSUnderlineStyle.RawValue) == NSUnderlineStyle.single.rawValue)
+        }
+
+        XCTAssertEqual(filteredAttributes.count, 2)
```

Comment:
You should test that `string` doesn't have these attributes. So create a closure with this filter logic and use it to check `filteredStringAtttributes.isEmpty`

Reply:
I've updated the test case. Could you check if this looks good? Thanks! 😄